docs/usage/forms.md

```diff
@@ -13,10 +13,40 @@ $post->fill([
 ]);
 ```
 
+## Request as array syntax
+
 To achieve this structure in your form - to prevent manipulating the form data just to save them. You can use the input name array `[]` [syntax](https://www.php.net/manual/en/faq.html.php#faq.html.arrays).
 
-```markup
+```html
 <input type="text" name="en[title]" />
 <input type="text" name="de[title]" />
 ```
 
+## Request as plain syntax
+
+If you want, you can structure the request inputs in colon notation too. Notice that with this approach the attribute goes first.
+
+```html
+<input type="text" name="title:en" />
+<input type="text" name="title:de" />
+```
+
+## Saving the request
+
+Both ways are ready to save the translatable attributes to the model without any other modification.
+
+```php
+public function update(Request $request, Post $post)
+{
+  $post->update($request->all());
+}
+```
+
+# Updating the default locale
+
+You can update the default locale attributes by accessing the translatable attributes from the model itself.
+
+```php
+$post->title = 'My first post'; // Will update the English translatable model
```

Comment:
A `fill()` line that uses an array with a simple title key would show that it also works with fill. 😊

Reply:
you mean `$post->fill(['title' => 'My first post']);`? Didn't know that! 🔥